cmd/cli/main.go

```diff
@@ -61,7 +61,7 @@ func init() {
 		if version == "" {
 			version = "undefined"
 		}
-		log.Printf("3scale-config-gen version is %s", version)
+		fmt.Printf("3scale-config-gen version %s\n", version)
```

Comment:
Why revert to `fmt` here but use `log` later?

Reply:
because this is an explicit action to output the version of the tool - it's not errors or extra/out-of-band information, like that of enabling fixups - makes sense?

Reply:
Ah I see commit message. Cool